### ISSUE TITLE
No gain found, shutting down | Planner not reachable

### ISSUE THREAD

**Issue by tiralonghipol** [ethz-asl/nbvplanner#21]
Hi.
After having explored all the area, the firefly stops middle air, reporting "Planner not reachable" in the terminal.
Is there any way to make the drone come back to the starting point ?

**Comment by birchera** [comment]
In the demo example the planner terminates after the exploration. You would need to implement a return to launch feature.

**Comment by tiralonghipol** [comment]
Thank you for the reply. 
Another thing: once the map is done and saved, is there any way to re-use it (maybe with another trajectory planner?) without 're-explore ' it ?
To be clear: with the first exploration the drone knows the map, and with a second fly it navigate into it.

**Comment by birchera** [comment]
Yes, the map is based on the octomap package and that supports both saving to and loading from binary files. So you could certainly save it after the exploration for later.
